src/ansys/fluent/core/workflow.py

```diff
@@ -756,6 +756,8 @@ def _assign(self, args: dict, fn) -> None:
         except Exception:
             old_state = None
         camel_args = {}
+        if "add_child" in args:
+            self._snake_to_camel_map["add_child"] = "AddChild"
```

Comment:
This is good for now if it resolves a test issue. Given that this seems to be specifying something thath looks like a generic mapping (surprisingly), would there be a different solution for the longer term?

Reply:
yes, @seanpearsonuk. I'll investigate further. Actually we should update the map during "_add_child()" call in compound task, but that was somehow not updating it. I'll look into it further.

Reply:
Added a TODO.